From b0469318657f14d961e38510a268cb0108718f71 Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Wed, 31 Aug 2016 21:21:16 +0200
Subject: [PATCH 1069/2241] arm: defconfig: mvebu: enable DSA and e6xxx switch
 family support

This config enables access to the network interfaces connected
via e6xxx switches, usin DSA linux infrastructure.

Change-Id: I45456c3c2c89f537fc2d71802c4880d555239c9e
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Signed-off-by: Neta Zur <neta@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33716
Tested-by: Star_Automation <star@marvell.com>
---
 arch/arm/configs/mvebu_v7_lsp_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/configs/mvebu_v7_lsp_defconfig b/arch/arm/configs/mvebu_v7_lsp_defconfig
index 702d127..271eec4 100644
--- a/arch/arm/configs/mvebu_v7_lsp_defconfig
+++ b/arch/arm/configs/mvebu_v7_lsp_defconfig
@@ -41,7 +41,9 @@ CONFIG_IP_PNP=y
 CONFIG_IP_PNP_DHCP=y
 CONFIG_IP_PNP_BOOTP=y
 CONFIG_BRIDGE=y
+CONFIG_NET_DSA=y
 CONFIG_VLAN_8021Q=y
+CONFIG_NET_SWITCHDEV=y
 CONFIG_BT=y
 CONFIG_BT_MRVL=y
 CONFIG_BT_MRVL_SDIO=y
@@ -75,6 +77,7 @@ CONFIG_MD_RAID1=y
 CONFIG_MD_RAID456=y
 CONFIG_NETDEVICES=y
 CONFIG_BONDING=y
+CONFIG_NET_DSA_MV88E6XXX=y
 CONFIG_E1000E=y
 CONFIG_IXGB=y
 CONFIG_IXGBE=y
-- 
2.7.4

